pkg/filesystem/subvolume.go

```diff
@@ -175,8 +177,19 @@ func listCephFSSubvolumes(ctx context.Context, clientsets *k8sutil.Clientsets, o
 			}
 			// append the subvolume which doesn't have any snapshot attached to it.
 			for _, sv := range subvol {
-				state := getSubvolumeState(ctx, clientsets, operatorNamespace, clusterNamespace, fs.Name, sv.Name, svg.Name)
+				state, err := getSubvolumeState(ctx, clientsets, operatorNamespace, clusterNamespace, fs.Name, sv.Name, svg.Name)
+				// subvolume info returns error in case of pending clone or if it is not ready
+				// it is suggested to delete the pvc before deleting the subvolume.
+				if err != nil {
+					if errors.Is(err, syscall.EAGAIN) {
+						logging.Warning("Found pending clone: %q", sv.Name)
+						logging.Warning("Please delete the pending pv if any before deleting the subvolume %s", sv.Name)
+						logging.Warning("To avoid stale resources, please scale down the cephfs deployment and then run the cleanup script.")
```

Comment:
What cleanup script?

Reply:
the subvolume delete command, maybe this could confuse the user, let me update.